wp-content/themes/core/components/section_nav/Section_Nav_Controller.php

```diff
@@ -103,11 +104,11 @@ public function __construct( array $args = [] ) {
 		$this->desktop_label     = (string) $args[ self::DESKTOP_LABEL ];
 		$this->more_label        = (string) $args[ self::MORE_LABEL ];
 		$this->sticky            = (bool) $args[ self::STICKY ];
-		$this->mobile_init_open  = (bool) $args[ self::MOBILE_INIT_OPEN ];
+		$this->mobile_init       = (string) $args[ self::MOBILE_INIT ];
```

Comment:
Any reason we can't keep it a bool? Checking a 'true'/'false' string is not ideal.

Reply:
The block registration is using a button_group for the options (Closed/Open), and that ACF field requires values I suppose I could reduce the return value from the Block's Model to be a boolean, rather than the actual ACF value. I'll update that per your suggestion.